src/extension.ts

```diff
@@ -177,6 +177,10 @@ export class Extension {
             //if this is a brightscript debug session
             if (e.type === 'brightscript') {
                 this.chanperfStatusBar.hide();
+                const config = e.configuration as BrightScriptLaunchConfiguration;
+                if (config.remoteControlMode && (config.remoteControlMode === true || config.remoteControlMode?.deactivateOnSessionEnd === true)) {
```

Comment:
Just check the `.deactivateOnSessionEnd`.
Probably something like `config?.remoteControlMode?.deactivateOnSessionEnd=== true` since our `DebugConfigurationProvider` will have sanitized this to always be in this structure.